app-arch/unar/unar-1.10.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs bash-completion-r1
+
+DESCRIPTION="unpacker for various archiving formats, e.g. rar v3"
+HOMEPAGE="https://unarchiver.c3.cx/"
+SRC_URI="
+	https://github.com/MacPaw/XADMaster/archive/v${PV}/XADMaster-${PV}.tar.gz
+	https://github.com/MacPaw/universal-detector/archive/1.1/universal-detector-1.1.tar.gz
+"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	media-sound/wavpack
+	app-arch/bzip2:=
+	dev-libs/icu:=
+	gnustep-base/gnustep-base
+	sys-libs/zlib:=
```

Comment:
Please sort this list alphabetically, and drop `:=` from zlib.

Reply:
I'm not quite sure if I understand this sub-slotting and rebuilding thing. `ldd` shows `/usr/bin/unar` being linked against `/lib64/libz.so.1`.

Shouldn't we rebuild `app-arch/unar` if `sys-libs/zlib` gets updated?

Reply:
When library **soname** changes, which it doesn't do with zlib. That subslot was mistakenly added and it's still causing confusion to this day...